supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -24,6 +187,264 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+Hidden replicas are primarily designed for services such as backups, bulk importing and exporting, or actions that require shutting down IdM services. Since no clients use a hidden replica, administrators can temporarily shut down the services on this host without affecting any clients.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:dns-srv-records[DNS SRV records]
+
+[discrete]
+[[host-system]]
+==== image:images/yes.png[yes] host system (noun)
+*Description*: The system on which the instrumentation modules, from SystemTap scripts, are compiled, to be loaded on target systems.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:target-system[target system]
+
+[discrete]
+[[id-mapping]]
+==== image:images/yes.png[yes] ID mapping (noun)
+*Description*: SSSD can use the SID of an AD user to algorithmically generate POSIX IDs in a process called _ID mapping_. ID mapping creates a map between SIDs in AD and IDs on Linux.
+
+* When SSSD detects a new AD domain, it assigns a range of available IDs to the new domain. Therefore, each AD domain has the same ID range on every SSSD client machine.
+* When an AD user logs in to an SSSD client machine for the first time, SSSD creates an entry for the user in the SSSD cache, including a UID based on the user's SID and the ID range for that domain.
+* Because the IDs for an AD user are generated in a consistent way from the same SID, the user has the same UID and GID when logging in to any Red Hat Enterprise Linux system.
+
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:id-ranges[id ranges], xref:sssd[SSSD]
+
+[discrete]
+[[id-ranges]]
+==== image:images/yes.png[yes] ID ranges (noun)
+*Description*: An ID range is a range of ID numbers assigned to the IdM topology or a specific replica. You can use ID ranges to specify the valid range of UIDs and GIDs for new users, hosts and groups. ID ranges are used to avoid ID number conflicts. There are two distinct types of ID ranges in IdM:
+
+* _IdM ID range_
++
+Use this ID range to define the UIDs and GIDs for users and groups in the whole IdM topology. Installing the first IdM server creates the IdM ID range. You cannot modify the IdM ID range after creating it. However, you can create an additional IdM ID range, for example when the original one nears depletion.
+
+* _Distributed Numeric Assignment (DNA) ID range_
++
+Use this ID range to define the UIDs and GIDs a replica uses when creating new users. Adding a new user or host entry to an IdM replica for the first time assigns a DNA ID range to that replica. An administrator can modify the DNA ID range, but the new definition must fit within an existing IdM ID range.
+
+
+Note that the IdM range and the DNA range match, but they are not interconnected. If you change one range, ensure you change the other to match.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:id-mapping[id mapping]
```

Comment:
This should be `[ID mapping]` per how the linked entry is capitalized

Reply:
+1